DiamondLightSource/mx-bluesky
# Add redis to murko forwarder

Fixes #570 

To test:
* Check it is understandable what the code is doing
* Confirm new unit tests work
